internal/util/paramtable/base_table.go

```diff
@@ -202,6 +203,32 @@ func (gp *BaseTable) Get(key string) string {
 	return value
 }
 
+func (gp *BaseTable) GetByPattern(pattern string) (map[string]string, error) {
+	formatter := func(key string) string {
+		ret := strings.ToLower(key)
+		ret = strings.ReplaceAll(ret, "/", "")
+		ret = strings.ReplaceAll(ret, "_", "")
+		ret = strings.ReplaceAll(ret, ".", "")
+		return ret
+	}
```

Comment:
duplicate code, extract it as a function

Reply:
> duplicate code, extract it as a function

Ok, but this func in package config was private, if we should change it to public?